lib/jormungandr/src/Cardano/Wallet/Jormungandr/Transaction.hs

```diff
@@ -56,6 +62,9 @@ newTransactionLayer (Hash block0) = TransactionLayer
     , estimateMaxNumberOfInputs =
         estimateMaxNumberOfInputsBase @t Binary.estimateMaxNumberOfInputsParams
 
+    , validateSelection = \(CoinSelection inps outs _) -> do
+        when (length inps > 255 || length outs > 255)
+            $ Left ErrExceededInpsOrOuts
```

Comment:
Same remark about code duplication applies here too. Plus, this check doesn't even occur at the moment in `mkStdTx`. 
Also, since we are starting to reference this `255` in various places in Jörmungandr, it'd be good to have them as named constant somewhere (probably in the `Binary` module as `maxNumberOfInputs` and `maxNumberOfOutputs`).

Reply:
added in jormungandr' Binary module both `maxNumberOfInputs` and `maxNumberOfOutputs`